pandas_transform/pandas_transform.md

```diff
@@ -0,0 +1,990 @@
+long_description: This is a longer description, which should be understandable for a lay audience. It will print under "Is this module right for me?" in the overview.
+estimated_time: 1 hour
+
+@learning_objectives  
+
+After completion of this module, learners will be able to:
+
+- Import `pandas` and use functions from the `pandas` package.
+- Load data into a `pandas` DataFrame.
+- Use the `.loc` method to explore the contents of a DataFrame
+- Filter a DataFrame using conditional statements.
+- Transform data in a DataFrame.
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+script: https://sagecell.sagemath.org/static/embedded_sagecell.js
+
+@sage
+<script input="hidden">
+// Make *any* div with class 'python' a Sage cell
+sagecell.makeSagecell({inputLocation: 'div.python',
+                       evalButtonText: 'Run python',
+                       languages: ["python"],
+                       hide: ['fullScreen', 'permalink'],
+                       });
+// Make *any* div with class 'python_run' a Sage cell
+sagecell.makeSagecell({inputLocation: 'div.python_run',
+                      evalButtonText: 'Run python',
+                      languages: ["python"],
+                      hide: ['fullScreen', 'permalink'],
+                      autoeval: 'true'
+                      });
+// Make *any* div with class 'python_link' a Sage cell
+sagecell.makeSagecell({inputLocation: 'div.python_link',
+                      evalButtonText: 'Run python',
+                      languages: ["python"],
+                      hide: ['fullScreen', 'permalink'],
+                      autoeval: 'false',
+                      linked: 'true'
+                      });
+// Make *any* div with class 'python_data_init' a Sage cell
+sagecell.makeSagecell({inputLocation: 'div.python_data_init',
+                      evalButtonText: 'Run python',
+                      languages: ["python"],
+                      editor: 'codemirror-readonly',
+                      hide: ['fullScreen', 'permalink','output','evalButton'],
+                      autoeval: 'true',
+                      linked: 'true',
+                      linkKey: "data"
+                      });       
+// Make *any* div with class 'python_data' a Sage cell
+sagecell.makeSagecell({inputLocation: 'div.python_data',
+                      evalButtonText: 'Run python',
+                      languages: ["python"],
+                      hide: ['fullScreen', 'permalink'],
+                      autoeval: 'false',
+                      linked: 'true',
+                      linkKey: "data"
+                      });                
+</script>
+@end
+-->
+
+# Using `pandas` for Tabular Data in python
+
+<div class = "overview">
+## Overview
+
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:** @estimated_time
+
+**Pre-requisites**
```

Comment:
Links in the prereqs will need to be updated

Reply:
Not just here, all of the links to the covid data will need to be updated!!!